src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -155,38 +145,33 @@ const FiatDetailsScreen = ({ route, navigation }: Props) => {
         ...implicitBody,
       }
 
-      await addNewFiatAccount(providerURL, fiatAccountSchema, completeBody)
-        .then((data) => {
-          // TODO Tracking here
-          dispatch(showMessage(t('fiatDetailsScreen.addFiatAccountSuccess')))
-        })
-        .catch((error) => {
-          // TODO Tracking here
-          if (error === FiatConnectError.ResourceExists) {
-            dispatch(showError(ErrorMessages.ADD_FIAT_ACCOUNT_RESOURCE_EXIST))
-          } else {
-            dispatch(showError(t('fiatDetailsScreen.addFiatAccountFailed')))
-          }
-          Logger.error(TAG, `Error adding fiat account: ${error}`)
-        })
+      // await addNewFiatAccount(quote.getProviderBaseUrl(), fiatAccountSchema, completeBody)
```

Comment:
not sure I understand why this is commented out now

Reply:
Because its unimplemented and always throws https://github.com/valora-inc/wallet/blob/main/src/fiatconnect/index.ts#L179

Reply:
Can't get to the review screen if it  throws